lib/atomic/accounts/user.ex

```diff
@@ -79,6 +81,17 @@ defmodule Atomic.Accounts.User do
     |> unique_constraint(:email)
   end
 
+  defp validate_handle(changeset) do
+    changeset
+    |> validate_required([:handle])
+    |> validate_format(:handle, ~r/^[a-zA-Z0-9_.]+$/,
+      message: "must only contain alphanumeric characters, numbers, underscores and periods"
```

Comment:
```suggestion
      message: gettext("must only contain alphanumeric characters, numbers, underscores and periods")
```
Maybe it is `Gettext.gettext` 😅